languages/backupwordpress-en_AU.po

```diff
@@ -0,0 +1,1211 @@
+# Copyright (C) 2015 Human Made Limited
```

Comment:
I'm not sure what purpose this file serves, it's a blank translation to Australian English? If you were trying to update the original English strings then you need to update the `.pot` file rather than adding a `.po` and `.mo`